src/libs/PopUpNotification.js

```diff
@@ -0,0 +1,138 @@
+import React, {
+    forwardRef, useImperativeHandle, useRef, useState,
+} from 'react';
+import {
+    StyleSheet, Text, View, Animated, Platform,
+} from 'react-native';
+import {
+    Directions, FlingGestureHandler, State, TouchableWithoutFeedback,
+} from 'react-native-gesture-handler';
+import colors from '../styles/colors';
+import Icon from '../components/Icon';
+import {Checkmark, Exclamation} from '../components/Icon/Expensicons';
+import ScreenWrapper from '../components/ScreenWrapper';
+import styles from '../styles/styles';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../components/withWindowDimensions';
+
+const desktopContainerStyle = {
+    maxWidth: '380px',
+    top: '20px',
+    right: 0,
+    position: 'fixed',
+};
+
+const popupStyles = StyleSheet.create({
+    container: {
+        flex: 1,
+        justifyContent: 'flex-start',
+        position: 'absolute',
+        width: '100%',
+        zIndex: 2,
+        ...Platform.select({
+            web: desktopContainerStyle,
+            macos: desktopContainerStyle,
+            windows: desktopContainerStyle,
+        }),
+    },
+    smallScreenWidth: {
+        maxWidth: 'none',
+    },
+    box: {
+        backgroundColor: colors.dark,
+        borderRadius: 8,
+        paddingVertical: 20,
+        paddingHorizontal: 30,
+        alignItems: 'center',
+        flexDirection: 'row',
+        justifyContent: 'space-between',
+        shadowColor: '#000',
+        shadowOffset: {
+            width: 0,
+            height: 2,
+        },
+        shadowOpacity: 0.25,
+        shadowRadius: 4,
+        elevation: 5,
+    },
+    bodyText: {
+        color: '#fff',
+        fontSize: 16,
+    },
+});
+
+const types = {
+    success: {
+        icon: Checkmark,
+        iconColor: colors.green,
+    },
+    error: {
+        icon: Exclamation,
+        iconColor: colors.red,
+    },
+    warning: {
+        icon: Exclamation,
+        iconColor: colors.orange,
```

Comment:
Where is this color coming from?

Reply:
`colors.js` is missing orange color, so i added one.`orange: '#e67e22',`

reason: for warning icon.

Reply:
Could you please use `#FED607` as the color (yellow instead of orange) and add it as a new color value for `colors.yellow`

Reply:
Looks good, thanks!